static/src/javascripts/projects/common/modules/check-mediator.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+
+import checks from './check-mediator-checks';
```

Comment:
Importing the checks is good, this means we can remove the _testRegisterCheck function and mock check-mediator-checks on the test.